ButtonA: 0
ButtonB: 1
ButtonX: 2
ButtonY: 3
ButtonLB: 4
ButtonRB: 5
ButtonSelect: 6
ButtonStart: 7
ButtonLeftThumb: 9
ButtonRightThumb: 10
AxisLeftThumbX: 0
AxisLeftThumbY: 1
AxisRightThumbX: 3
AxisRightThumbY: 4
AxisLeftTrigger: 2
AxisRightTrigger: 5
AxisDPadX: 6
AxisDPadY: 7
